vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/AbsMessageLocationItem.kt

```diff
@@ -75,6 +75,7 @@ abstract class AbsMessageLocationItem<H : AbsMessageLocationItem.Holder> : AbsMe
         GlideApp.with(holder.staticMapImageView)
                 .load(location)
                 .apply(RequestOptions.centerCropTransform())
+                .placeholder(holder.staticMapImageView.drawable)
```

Comment:
This fix seems to come from this suggestion: https://github.com/bumptech/glide/issues/2505#issuecomment-532168345

It's not recommended to use that in RecyclerView, which is the case here (see the comments below).

I should work, but may lead to some issue.

Not a blocker anyway.

Reply:
> This fix seems to come from this suggestion: https://github.com/bumptech/glide/issues/2505#issuecomment-532168345

Exactly, I couldn't find any other solutions. I could understand the below comment if we were setting the actual source but we are just setting the placeholder.

+ We are going to have a test session next week.

Reply:
I'm agree with @bmarty about this fix. Further tests are needed.